### DISCUSSION TITLE
Keep Color Library Panel Open

### DISCUSSION THREAD

**Discussion by matthewfarlymn** [rsayfutdinov/Pikka#35] [Q&A]
Is it possible to keep the color library panel open when selecting colors for a palette? It's a little irritating that I need to continue re-opening the color library when using the color selection feature. If this isn't an option it would be a lovely addition to the app.

**Comment by rsayfutdinov** [comment]
Take a look at the preferences. There is a checkbox "Dismiss library on mouse click". 
I think you just need to turn it off.

<img width="836" alt="Screenshot 2022-05-23 at 21 10 39" src="https://user-images.githubusercontent.com/12187524/169889898-1410076f-b87a-4d1d-82e6-9f13af4b3fc7.png">

**Reply by matthewfarlymn** [reply]
I have that disabled, which works except when I want to select a color to add and the "flyout" disappears rather than keeping me in the pallet.

https://user-images.githubusercontent.com/5617870/172483072-bfead852-d68a-43cb-adcb-a5d1046d5e6e.mov

**Reply by rsayfutdinov** [reply]
I got it! 
An update is coming :)

**Reply by matthewfarlymn** [reply]
Thank you :)

**Reply by matthewfarlymn** [reply]
@rsayfutdinov I saw the update and it's working now! Thank you very much!
